### DISCUSSION TITLE
Pypi

### DISCUSSION THREAD

**Discussion by kulki007** [ib-api-reloaded/ib_async#1] [General]
What steps need to be done to make this project available on pypi? I can help with anything that you might need.

**Comment by mattsta** [comment]
Was able to get around to the package creation this evening.

Now we have https://pypi.org/project/ib_async/ and an officially released `v1.0.0` which is mostly naming and formatting changes (no added features or bug fixes).

**Reply by salsasepp** [reply]
Very grateful for your initiative and the work you're putting in. I'm switching over my IB data-gathering to ib_async today, will report any gotchas (none so far).

**Reply by tong-bai** [reply]
I have just saw the news and found this. Thank you very much for your effort. I would have offered to help maintaining the code but I am not a real programmer. Dealing with IB's API itself is beyond my capability but I am an experienced trader with deep understanding of financial market. Please let me know if I am able to help with anything.
